src/transform/bare-specifiers.ts

```diff
@@ -29,7 +29,7 @@ export const bareToPathSpecifiersTransform = (): Transform =>
       const scriptSource = await getFileContents(file);
 
       try {
-        const pathParts = file.path.split('node_modules/');
+        const pathParts = file.path.split('node_modules');
```

Comment:
#7 Is this log helpful? Maybe we need to add some kind of verbosity flag?

Reply:
is this not used anywhere else then in the login ?
This that is not the case, then it was only to fix the log output, and avoid misleading users